pages.pt_BR/osx/indent.md

```diff
@@ -0,0 +1,12 @@
+# indent
+
+> Altera a aparência de um programa C/C++ inserindo ou excluindo espaços em branco.
+> Mais informações: <https://www.freebsd.org/cgi/man.cgi?query=indent>.
+
+- Formata código fonte C/C++ de acordo com o estilo Berkeley:
+
+`indent {{caminho/para/fonte.c}} {{caminho/para/fonte_identado.c}} -nbad -nbap -bc -br -c33 -cd33 -cdb -ce -ci4 -cli0 -di16 -fc1 -fcb -i4 -ip -l75 -lp -npcs -nprs -psl -sc -nsob -ts8`
```

Comment:
```suggestion
`indent {{caminho/para/fonte.c}} {{caminho/para/fonte_identado.c}} -nbad -nbap -bc -br -c33 -cd33 -cdb -ce -ci4 -cli0 -di16 -fc1 -fcb -i4 -ip -l75 -lp -npcs -nprs -psl -sc -nsob -ts8`
```

Warning (no fix available): too many short options can be converted to long ones.
Note: style guide should such case not to make commands too long after short->long option conversion.

Reply:
Can we keep it this way? 
(I couldn't find the long counterpart)

Reply:
Yes, we can because I can't enforce you to follow unwritten rules in our style guide. 😄